src/client/css/partials/user-menu.css

```diff
@@ -0,0 +1,120 @@
+.user-menu-wrapper {
+  font: normal 14px/1.5 Inter, Inter-fallback, sans-serif;
```

Comment:
I often waffle between px and rem... for fonts we're so far using rem throughout (I believe) – should we stay consistent?

Reply:
Yes, let’s stay consistent. I converted the values to `rem`.